src/Symfony/Component/Routing/Loader/Psr4DirectoryLoader.php

```diff
@@ -88,4 +90,17 @@ private function loadFromDirectory(string $directory, string $psr4Prefix): Route
 
         return $collection;
     }
+
+    private function validateNamespace(mixed $namespace): void
+    {
+        if (str_contains($namespace, '/')) {
```

Comment:
Not sure we need a special case for `/`.

Reply:
I've added it to be more precise about the error message, and to help the user as much as possible.

Reply:
I agree with @fabpot here, saying that a namespace part is not valid is enough for me, because it'll show that it contains a `/` and lead the user to the issue.

Reply:
Well, `/` is not part of the namespace, it's a "bad" separator.

But I think this will be a no-subject after https://github.com/symfony/symfony/pull/59189#discussion_r1881561397